CRM/Contribute/BAO/Contribution.php

```diff
@@ -2098,7 +2098,7 @@ public function loadRelatedObjects(&$input, &$ids, $required = FALSE, $loadAll =
         'payment_processor'
       );
       if ($paymentProcessorID) {
-        $intentionalEnotice = $CRM16923AnUnreliableMethodHasBeenUserToDeterminePaymentProcessorFromContributionPage;
+        //$intentionalEnotice = $CRM16923AnUnreliableMethodHasBeenUserToDeterminePaymentProcessorFromContributionPage;
```

Comment:
Can we leave this enotice in - anything that hits an enotice on this line should trigger a bug report as it is dodgier than a clown in a dodgem